### DISCUSSION TITLE
Segment and OAL Endpoint

### DISCUSSION THREAD

**Discussion by BritneyStars** [apache/skywalking#8410] [Q&A]
I find the OAL metrics about the scope endpoint is not including some endpoints in segment indice. such as  contain Kafka keywords and Redis.  I want to kown to   how to identify the actual endpoint and component endpoint such as Kafka 、Redis 、eureka and so on in OAL calculate

**Comment by wu-sheng** [comment]
Do you read the document about what is the endpoint?

**Reply by BritneyStars** [reply]
ok！I just find some endpoint_id in segment indice not found in endpoint oal metrics indices.

**Reply by wu-sheng** [reply]
They are different things. Read the doc.
